src/Microsoft.Extensions.Configuration.Abstractions/IConfigurationBuilder.cs

```diff
@@ -26,11 +26,11 @@ public interface IConfigurationBuilder
         /// </summary>
         /// <param name="source">The configuration source to add.</param>
         /// <returns>The same <see cref="IConfigurationBuilder"/>.</returns>
-        IConfigurationBuilder Add(IConfigurationSource source);
+        new IConfigurationBuilder Add(IConfigurationSource source);
```

Comment:
I took a look at getting rid of Sources in this PR, and that caused a bunch of breaking changes, didn't seem worth going down that path at all...